configure.py

```diff
@@ -97,6 +97,9 @@ def is_solaris(self):
     def is_aix(self):
         return self._platform == 'aix'
 
+    def is_aix_os400_variant(self):
+        return self._platform == 'aix' and os.uname().sysname.startswith('OS400')
```

Comment:
Please check for `self._platform` is 'os400' first, since at some point we'll fix Python to not lie about what platform it's on.